srcpkgs/hplip-plugin/template

```diff
@@ -0,0 +1,61 @@
+# Template file for 'hplip-plugin'
+# Must be in sync with main hplip package
```

Comment:
also add this notice to the hplip package

Reply:
If we note this is the main hplip package, it should probably be a bit milder:

> When possible, keep this in sync with hplip-plugin.

In case there are some troubles updating the plugin, we might not want to deter people from updating the main package.

Reply:
If changed the comment to `# When possible, keep this in sync with hplip-plugin.`

Reply:
I will test a out of sync version of the plugin later this week; as I remember some time ago it would not let me scan without the update.